packages/editor/src/store/test/effects.js

```diff
@@ -257,17 +268,7 @@ describe( 'effects', () => {
 
 			handler( { post, previousPost, postType }, store );
 
-			expect( dispatch ).toHaveBeenCalledTimes( 1 );
-			expect( dispatch ).toHaveBeenCalledWith( expect.objectContaining( {
-				notice: {
-					content: <p>Post published.{ ' ' }<a>View post</a></p>, // eslint-disable-line jsx-a11y/anchor-is-valid
-					id: 'SAVE_POST_NOTICE_ID',
-					isDismissible: true,
-					status: 'success',
-					spokenMessage: 'Post published.',
-				},
-				type: 'CREATE_NOTICE',
-			} ) );
+			expect( dataDispatch( 'core/notices' ).createSuccessNotice ).toHaveBeenCalled();
```

Comment:
Is there any reason why we can assert using more detailed check similar to what was previousle and what I see later in the file, e.g.:
```js
expect( dataDispatch( 'core/notices' ).createErrorNotice ).toHaveBeenCalledWith( 'Publishing failed', { id: SAVE_POST_NOTICE_ID } );
```

Reply:
> Is there any reason why we can assert using more detailed check similar to what was previousle and what I see later in the file, e.g.:
> 
> ```js
> expect( dataDispatch( 'core/notices' ).createErrorNotice ).toHaveBeenCalledWith( 'Publishing failed', { id: SAVE_POST_NOTICE_ID } );
> ```

Updated in rebased c5985282b2a2b977fb10ddb37eb8a797fa13e566.